### DISCUSSION TITLE
Problem with relative links in popup of changing images.

### DISCUSSION THREAD

**Discussion by tarzinio** [tinymce/tinymce#8836] [General]
Hi

I have set the link change to relative, but after inserting it via an external filemanager, and then opening the photo change popup, the link remains absolute. Only after saving it changes it to relative. strangely, in the code preview, the link is correct after uploading.
![obraz](https://github.com/tinymce/tinymce/assets/29274594/994a83cd-f0d2-4aa2-acdf-579ebe4a144f)
![obraz](https://github.com/tinymce/tinymce/assets/29274594/f65c8bce-1490-42fe-bf25-653834f7da1d)
![obraz](https://github.com/tinymce/tinymce/assets/29274594/3bddc81d-5009-448a-a32a-32f05cf44fe7)

Does anyone know how to change it?

Version: 6.5.0
Browser: Firefox (with last update)

**Comment by ranwenqi** [comment]
How to update in real time if there are changes?

**Comment by tarzinio** [comment]
@ranwenqi 
I am sending a video with which I show what I mean.

https://github.com/tinymce/tinymce/assets/29274594/1dfc6eac-1fd2-4a22-889a-4766c3127302
